# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/planner/hotfix-shared.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- planner-0.14.92/meson.build.vanilla	2025-10-15 15:52:58.383127922 +0200
+++ planner-0.14.92/meson.build	2025-10-15 15:53:05.784192749 +0200
@@ -86,7 +86,7 @@
 subdir('tests')
 subdir('docs')
 
-meson.add_install_script('meson_post_install.sh')
+#meson.add_install_script('meson_post_install.sh')
 
 status = [
   '', '',
